x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/RestEnrichTestCase.java

```diff
@@ -148,7 +148,7 @@ public void testNonExistentEnrichPolicy() throws IOException {
         );
         assertThat(
             EntityUtils.toString(re.getResponse().getEntity()),
-            containsString("unresolved enrich policy [countris], did you mean [countries]?")
+            containsString("enrich policy [countris] doesn't exist, did you mean [countries]?")
```

Comment:
As there might a security setting (thus the policy can exist but not be visible) might be better to have the "cannot find enrich policy [countries], did you mean [...]).

Reply:
It can happen in the future, but the current security model is all or nothing for enrich policies. I will look into it later, as I am working on the coordinator mode based on this PR.